ci-operator/step-registry/ipi/install/install/stableinitial/ipi-install-install-stableinitial-ref.yaml

```diff
@@ -8,8 +8,6 @@ ref:
       memory: 2Gi
   dependencies:
   - name: "release:latest"
-    env: OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE
```

Comment:
@vrutkovs , why was this not defaulting to `stable:initial`?

Reply:
`stable` imagestream doesn't have a release imagetag, but I'm fine with defaulting this to `release:initial`